recipes/linaro_forge/25.1/modules.yaml

```diff
@@ -12,12 +12,12 @@ modules:
     arch_folder: false
     # Where to install modules
     roots:
-      tcl: /user-tools/modules
+      tcl: /user-environment/modules
```

Comment:
The uenv is installed at `/user-tools`, so shouldn't the path be set to `/user-tools`?

I think that stackinator will actually modify this for you, so the change probably has no impact.
https://github.com/eth-cscs/stackinator/blob/main/stackinator/recipe.py#L319